components/ILIAS/Authentication/classes/class.ilAuthPageEditorGUI.php

```diff
@@ -199,8 +284,13 @@ private function handleLogoutPageActions(): void
      */
     private function getLangKeysToUpdate(): array
     {
+        $request_value = match ($this->context) {
```

Comment:
Again, we could/ should think about moving this into the `Enum`, like the language variable retrieval.